resource_booking_simultaneous_human/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 Coop IT Easy SC
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Resource booking simultaneous availability for humans",
+    "summary": """
+        When a resource of type human is scheduled for a booking, they remain
+        simultaneously available for other bookings.""",
+    "version": "14.0.1.0.0",
+    "category": "Appointments",
+    "website": "https://github.com/OCA/calendar",
+    "author": "Odoo Community Association (OCA), Coop IT Easy SC",
+    "maintainers": ["carmenbianca"],
+    "license": "AGPL-3",
+    "application": False,
+    "depends": [
+        "resource_booking",
+    ],
+    "excludes": [],
+    "data": [],
+    "demo": [],
+    "qweb": [],
```

Comment:
avoid empty keys (see [here](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#11modules)). i would also remove the `application` key, as `False` is the default value.

Reply:
Done